bigframes/session/__init__.py

```diff
@@ -832,9 +834,13 @@ def _read_gbq_table(
                     ordering=ordering,
                 )
             else:
-                array_value = self._create_total_ordering(table_expression)
+                array_value = self._create_total_ordering(
+                    table_expression, table_size=table.num_rows
```

Comment:
Could you please try this with a VIEW type table? I don't think `num_rows` is always populated.

Reply:
yes, this won't be populated for views or potentially with streaming inserts. defaulting to assume table is large and need 128 bits.